test/e2e/dnslookup_test.go

```diff
@@ -32,13 +32,14 @@ import (
 	"testing"
 	"time"
 
+	"golang.org/x/sys/windows"
```

Comment:
The imports here look wong — only the `github.com/rancher-sandbox/` imports should be in a separate section.